test/integration/legacy_routes/handler_test.rb

```diff
@@ -0,0 +1,130 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2008-2017, Puzzle ITC GmbH. This file is part of
+#  Cryptopus and licensed under the Affero General Public License version 3 or later.
+#  See the COPYING file at the top-level directory or at
+#  https://github.com/puzzle/cryptopus.
+require 'pry'
+require 'test_helper'
+
+class LegacyRoutes::UrlTest < ActionDispatch::IntegrationTest
+  include IntegrationTest::DefaultHelper
+
```

Comment:
why not add a comment above test to explain how this works ....
e.g. \# /de/teams/42/groups/43/accounts/42 -> /accounts/42
then you can use a pretty short name for the test case ... test 'legacy groups url'

a legacy route always includes the locale !